src/registrar/models/transition_domain.py

```diff
@@ -5,14 +5,18 @@
 
 class StatusChoices(models.TextChoices):
     READY = "ready", "Ready"
-    HOLD = "hold", "Hold"
+    ON_HOLD = "on hold", "On Hold"
 
 
 class TransitionDomain(TimeStampedModel):
     """Transition Domain model stores information about the
     state of a domain upon transition between registry
     providers"""
 
+    StatusChoices = StatusChoices
+
+    StatusChoices = StatusChoices
```

Comment:
merge conflict? why is it here twice?

Reply:
Wow, looks like it.  I wonder why it didn't throw any errors.  Removing duplicate...